src/db/tables/featureFlags.ts

```diff
@@ -68,9 +68,12 @@ export async function getEnabledFeatureFlags(
     .and.where("is_enabled", true);
 
   if (!options.ignoreAllowlist) {
-    query = query.and
-      .whereRaw("ARRAY_LENGTH(allow_list, 1) IS NULL")
-      .orWhereRaw("? = ANY(allow_list)", options.email);
```

Comment:
Just to clarify: this changes the query from `(x AND y) OR z` to `x AND (y OR z)`, the latter being what we intended.

(`y` = allowlist is empty, `z` = the user's email address is in the allowlist.)